tests/CMakeLists.txt

```diff
@@ -43,6 +44,15 @@ if(HAVE_GET_CURRENT_CGROUP_ID)
 endif(HAVE_GET_CURRENT_CGROUP_ID)
 target_link_libraries(bpftrace_test arch ast parser resources)
 
+find_library(BPF_LIB bpf)
```

Comment:
Isn't this covered by `find_package(LibBpf)` in the base `CMakeLists.txt`?

Reply:
oops.. right, I'll remove it